### PR TITLE
Fix LangtonSubtest's JavaDoc; add noZombieAntsTest with speedup=10

### PR DESCRIPTION
**Pull request by pcworld**

### CODE REVIEW

File: src/final2/subtests/MoveTest.java [hunk 1]
```diff
@@ -276,6 +276,7 @@ public void terminateAfterAllHaveLeftTest() {
 			move(1)
 		};
 		sessionTest(runs, Input.getFile(inputFile), "rule=90-90-90-270-90", "speedup=9");
+		sessionTest(runs, Input.getFile(inputFile), "rule=90-90-90-270-90", "speedup=10");
```
Comment (jGleitz) [inline src/final2/subtests/MoveTest.java:279]: make it 20 / 50 / 100 to have more discarded moves?

**Comment by jGleitz** [comment]
lgtm

**Comment by jGleitz** [comment]
Merging in favour of #172. Please don't stop the discussion.

File: src/final2/subtests/MoveTest.java (hunk 1 above)
Comment (pcworld) [inline src/final2/subtests/MoveTest.java:279]: all three or would 100 suffice?

**Pull request merged**